src/components/Home.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaMicrophone } from 'react-icons/fa';
+import { FiSettings } from 'react-icons/fi';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { fetchCurrencies, forexPair } from '../redux/ForexSlice';
+import '../styles/home.css';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const { currencyData, isLoading, error } = useSelector((state) => state.currencies);
+
+  useEffect(() => {
+    dispatch(fetchCurrencies());
+  }, [dispatch]);
+
+  const handleClicked = (ticker) => {
+    dispatch(forexPair(ticker));
+  };
+
+  const dateOptions = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: false,
+  };
+  const today = new Date();
+  const formattedDate = new Intl.DateTimeFormat('en-US', dateOptions).format(today);
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return (
+      <div>
+        Error:
+        {error.message}
+      </div>
+    );
+  }
+
+  return (
+    <div className="list-container">
+      <div className="recent-container">
+        <p className="recent-p">most recent price</p>
+        <div className="recent-icons">
+          <FaMicrophone />
+          <FiSettings />
+        </div>
+      </div>
+      <div className="todays-rate-container">
+        <h1>Today&apos;s Rates</h1>
+        <p>147 pairs</p>
+        <span>{formattedDate}</span>
+      </div>
+      <div className="opening-prices">
+        <p>OPENING PRICES</p>
+      </div>
+      <ul className="pairs-container">
+        {currencyData && currencyData.map((currency) => (
+          <li className="individual-pair" key={currency.ticker}>
+            <Link to="/ForexDetails" onClick={() => handleClicked(currency.ticker)}>
+              <div className="right-arrow">
+                <p>
+                  <BsArrowRightCircle />
+                </p>
+              </div>
+              <div className="ticker-name">
+                <p>
+                  {currency.ticker}
+                </p>
+              </div>
+              <div className="ticker-rate">
+                <p>
+                  Open:
+                  {currency.open}
+                </p>
+              </div>
+            </Link>
+          </li>
+        ))}
```

Comment:
You did well on this project. 👏🏾 Beautiful and clean code 👏🏾 

However, I see one feature is missing, there should be a way for users to filter/search the currency pairs on the home page without scrolling to find any particular pair. This is a requirement for this project. It will improve the overall **_ease of use_** of your application. 

See [example here](https://crypto-hub-247.netlify.app/) for ideas.
--- |
![image](https://user-images.githubusercontent.com/84156714/233610166-2c4e4f65-ac7d-4a7b-9c9d-03c62bbb505b.png) |
Click [here ](https://github.com/bobb-Rob/Crypto-Hub/blob/dev/src/components/CryptoContainer.js) to see the code to implement this.  This is just one way to do this, there are other methods, feel free to solve this your own way. 

![image](https://user-images.githubusercontent.com/84156714/233608211-19470a4e-a73a-48d9-bb08-0d0c07d22cb6.png)

Reply:
Thank you for this, I am on it 👍